metacatalog/db/views/locations.sql

```diff
@@ -1,4 +1,4 @@
---CREATE OR REPLACE VIEW locations_realdata AS
+--CREATE OR REPLACE VIEW locations AS
 select 
 	st_asewkt(t.point_location),
```

Comment:
do we need this as WKT? 
Also, I would rename the resulting column, otherwise it will be named `st_asewkt`, which is kind of annoying.

Reply:
You are right, we do not really need that column, as it is the same as column `point_location` but as WKT.

I would just drop that column, as it is redundant information.